test/functorch/common_utils.py

```diff
@@ -378,6 +379,15 @@ def wrapped(fn):
     return wrapped
 
 
+def skipOpsIfTorchDynamo(test_case_name, base_test_name, to_skip):
+    if TEST_WITH_TORCHDYNAMO:
+        return skipOps(test_case_name, base_test_name, to_skip)
+
+    def wrapped(fn):
+        return fn
+    return wrapped
```

Comment:
Do we use this anywhere in this PR?

If not, then, we should remove this. I'd prefer the mechanism to be per-op instead of a new decorator, so that one can insert something like `skip('index_reduce', if=TEST_WITH_TORCHDYNAMO)` into one of the skip/xfail lists.